core/admin/mailu/models.py

```diff
@@ -38,6 +38,19 @@ def process_result_value(self, value, dialect):
         return filter(bool, value.split(","))
 
 
```

Comment:
We should maybe parametrize the type with a length for the underlying string, so that alembic properly handles the table creation.

Reply:
I moved the parameters into the class IdnaDomain and IdnaEmail in 7f5bd98a2e5bd94f3ac128a05d838a9b98d5b9d6